README-CONCLUSION.md

```diff
@@ -0,0 +1,43 @@
+# Conclusion
+
+Observability signals like logs, metrics, and traces are what allows us as developers and operators to understand the behavior of our applications. The need to observe serverless functions is no different than other applications. For better or worse, not having access to the execution environment forces a degree of rigor. Though initially painful, it's ultimately a good thing.
```

Comment:
> Though initially painful, it's ultimately a good thing.

I'd challenge this phrasing. This is a mental shift to design for good observability that will make you a better developer and enhance the quality of your serverless applications.